packages/svelte/src/internal/client/dom/elements/attributes.js

```diff
@@ -368,7 +368,10 @@ export function set_attributes(
 				} else if (key === 'checked') {
 					/**@type {HTMLInputElement}*/ (element).defaultChecked = default_checked_reset;
 				}
-			} else if (setters.includes(name) && (is_custom_element || typeof value !== 'string')) {
+			} else if (
+				is_default_value_or_checked ||
```

Comment:
do we need this check? AFAIK the setters should include defaultValue/defaultChecked

Reply:
Before this it was not working at all so I assumed no (tbh I didn't check)

Reply:
`setters` _does_ include those values, but `defaultValue` fails the `typeof value !== 'string'` part of the check